dim_checks/moz-fx-data-shared-prod/telemetry/cohort_daily_statistics/dim_checks.yaml

```diff
@@ -0,0 +1,88 @@
+dim_config:
+  owner:
+    email: lvargas@mozilla.com
```

Comment:
Kik, what does it mean that I am the owner and who will receive notifications?

Reply:
no, this is purely so that we know who owns this dataset. In we detect the issues and this should be the default first contact person to ask about the issues found.